src/studio/nuxeo-studio/working-in-studio/branding.md

```diff
@@ -391,168 +392,93 @@ history:
-
-With a web designer's help, you can have many other possibilities.
-
-### Additional Options
-
-<div class="table-scroll"><table class="hover"><tbody><tr><th colspan="1">Login page layout</th><th colspan="1">Studio Configuration</th><th colspan="1">Role</th></tr><tr><td colspan="1">![]({{file name='login_layout.png'}} ?w=300,border=true,thumbnail=true)</td><td colspan="1">
-
-![]({{file name='login_remove_boxes.png'}} ?w=310,border=true)
-
-</td><td colspan="1">To let you customize the login screen completely, the news and footer can be removed.</td></tr><tr><td colspan="1">
-
-![]({{file name='login_box.png'}} ?w=310,border=true)
-
-</td><td colspan="1">
-
-If you have a simple color used as the background image in the login page, you can add a color to the login box and to the button.
-
-This option must be used carefully: don't use heavy patterns or things that will make the readability difficult or hamper the interpretation. The login box style must stay very simple and light to keep the login action as a user-friendly step.
-
-</td></tr></tbody></table></div>
-
-{{! /multiexcerpt}}
-
-{{> anchor 'css'}}
-
-## CSS Editor Tab
-
-The CSS Editor is dedicated to web designers who know CSS and its syntax. It's the place to declare your new CSS rules or override existing rules that will be used into the Nuxeo main tabs: Home, Document Management and Admin Center.
-
-{{#> callout type='info' }}
-
-Have a look at the [Nuxeo UI Style Guide](http://showcase.nuxeo.com/nuxeo/styleGuide/) to see all the existing CSS rules used by default in your platform, and understand more about readability, usability, layout design, and user experience.
+-   A picture covering all the screen. This is the strategy chosen by Nuxeo.
+-   A logo with a transparent background with a background color.
 
+{{#> callout type='warning' }} While selecting your background, be careful not to use heavy files to ensure a quick display to your users.
 {{/callout}}
 
-### Declaring a New CSS Class and Its Rules
-
-You can add style classes in your widgets in [forms]({{page page='form-layouts'}}), [content views]({{page page='content-views'}}), and [tab layout]({{page page='tabs'}}) to manage the rendering of your elements.
-
-Let's add a background color on a widget's container:
-
-<div class="table-scroll"><table class="hover"><tbody><tr><th colspan="1">
-
-1.  Widget Editor
-
-</th><th colspan="1">2\. CSS Editor</th><th colspan="1">3\. Result</th></tr><tr><td colspan="1">
-
-![]({{file name='blue_background_config.png'}} ?w=300,border=true,thumbnail=true)
+### Selecting a Background Image  
 
-</td><td colspan="1">
+1.  In `Background Image`, click on **Select Resources**.
+1.  Define options for the image, such as defining the position (in pixel or position) of the image or if the background cover all the screen (with CSS3 / recent browsers).
```

Comment:
Maybe a shorter sentence would be better?:
`Define options for the image, such as its position (in pixel or position), or if it should cover all the screen (with CSS3 / recent browsers`

Reply:
Good idea